.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: php
 
-sudo: false
-
 php:
-  - 5.6
+    - 7.2
+    - 7.3
+    - 7.4snapshot
```

Comment:
Turns out Travis doesn't have an image for php 7.4 yet. I added the snapshot image here and allowed it to fail below since I noticed test failures against it.